tests/ssp/host/host.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <cassert>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include "ssp_u.h"
+
+// uint64_t host_my_ocall(uint64_t val)
+//{
+//    return val * MY_OCALL_MULTIPLIER;
+//}
+
+// static bool g_func2_ok = false;
+//
+// void host_func2(const unsigned char*)
+//{
+//    g_func2_ok = true;
+//}
+
+// static oe_enclave_t* g_enclave = NULL;
+// static bool g_reentrancy_tested = false;
+// void host_test_reentrancy()
+//{
+//    oe_result_t result = enc_test_reentrancy(g_enclave);
+//    printf("result ==== %s\n", oe_result_str(result));
+//    OE_TEST(result == OE_REENTRANT_ECALL);
+//    g_reentrancy_tested = true;
+//}
+
+int main(int argc, const char* argv[])
+{
+    if (argc != 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE\n", argv[0]);
+        exit(1);
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    oe_enclave_t* enclave = NULL;
+    oe_result_t result = oe_create_ssp_enclave(
```

Comment:
Where is oe_create_ssp_enclave defined/implemented?  Right now I can't find it.   I would ask why ssp is in the function name, as opposed to using a flag or something.

Reply:
the oe_create_ssp_enclave() function will been auto generated by oeedger8r during the application compiling. Here ssp is not used as a flag but the name of test.